Duck.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Do you want to buy a duck?</title>
+  
+  <style>
+    body{
+        height: 100%;
+        width: 100%;
+        background-image: url("duckbackground.png");
+        background-repeat: repeat;
+    }
+    #content {
+        margin: auto;
+        height: 100%;
+        width: 90%;
+    }
+    form {
+        margin: auto;
+    }
+    #output {
+        overflow:auto;
+        background-color: rgba(255,255,255,0.7);
+    }
+  </style>
+  
+</head>
+<body>
+<div id="content">
+        <input type="text" id="numpeople" name="inputbox" placeholder="How many people are there?" style="width:175px" > 
+        <input class="enterbutton" type="Submit" value="Do you want to buy a duck?">
+    <div id="output">
+    </div>
+</div>
+</body>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.0/themes/smoothness/jquery-ui.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+<script>
+$( document ).ready(function() {
+var n;
+var words = [];
+
+wh = $(window).height();
+$('#output').css('height',wh*0.9+"px");
+
+
+$(".enterbutton").click(function() {
+    document.getElementById("output").innerHTML = "";
+    words = [];
+    n = document.getElementById("numpeople").value;
+    if(isNaN(n)) {
+        words.push("Please input an integar number of people.");
```

Comment:
Integer number reads a little awkwardly, maybe something like "Please enter a whole number" or something like that